composer.json

```diff
@@ -15,7 +15,7 @@
         "jean85/pretty-package-versions": "~1.5.0",
         "mglaman/phpstan-drupal": "^0.12.4",
         "nette/neon": "^3.1",
-        "phpstan/phpstan": "^0.12.26",
+        "phpstan/phpstan": "^0.12.42",
```

Comment:
@mglaman, would it make sense to drop this since its alreade required by mglaman/phpstan-drupal?

Reply:
Yeah, I suppose we can. The minimum supported phpstan/phpstan version  would be defined in mglaman/phpstan-drupal anyways